Sources/Target/POSIX/Thread.cpp

```diff
@@ -119,6 +119,8 @@ ErrorCode Thread::resume(int signal, Address const &address) {
                                    info, signal, address));
     _state = kRunning;
     _stopInfo.signal = 0;
+    _stopInfo.reason = StopInfo::kReasonNone;
```

Comment:
I know we wrote this when we were hacking things around on your tree, but I don't think we need this. Whoever uses the thread object should use `_stopInfo` if `_state` is `kRunning`.